source/codegen/metadata/nixnetsocket/enums.py

```diff
@@ -1,18 +1,144 @@
 enums = {
-  'OptName': {
+  'OperationalStatus': {
+      'enum-value-prefix': 'OPERATIONAL_STATUS',
+        'values': [
+            {
+                'name': 'DOWN',
+                'value': 0
+            },
+            {
+                'name': 'UP',
+                'value': 1
+            }
+        ]
+    },
+     'IPStackInffoStringFormat': {
+        'enum-value-prefix': 'IPSTACK_INFO_STR_FORMAT',
+        'values': [
+            {
+                'name': 'JSON',
+                'value': 0
+            },
+            {
+                'name': 'TEXT',
+                'value': 1
+            }
+        ]
+    },
+    'AddressFamilies': {
+        'values': [
+            {
+                'name': 'AF_UNSPEC',
```

Comment:
I think we should strip prefixes like 'AF_' from enum value names when their covered by the Enum's name (AddressFamilies covers AF_, ProtocolFamilies covers PF_, etc.)

Reply:
I've striped these off